src/components/withCurrentReportID.tsx

```diff
@@ -1,12 +1,10 @@
 import type {NavigationState} from '@react-navigation/native';
-import type {ComponentType, ForwardedRef, RefAttributes} from 'react';
-import React, {createContext, forwardRef, useCallback, useMemo, useState} from 'react';
-import getComponentDisplayName from '@libs/getComponentDisplayName';
+import React, {createContext, useCallback, useMemo, useState} from 'react';
```

Comment:
Isn't the idea to just get rid of this whole file? I only see one place using `updateCurrentReportID`, which could also be removed if I'm not mistaken.

Reply:
Yes, I think we'd want to move from the HOC to the hook fully. Let's migrate that code.